go.mod

```diff
@@ -25,4 +25,7 @@ replace (
 	github.com/keybase/go-keychain => github.com/99designs/go-keychain v0.0.0-20191008050251-8e49817e8af4
 	google.golang.org/grpc => google.golang.org/grpc v1.33.2
 	github.com/tendermint/tendermint => github.com/tendermint/tendermint v0.34.14
+	github.com/cosmos/cosmos-sdk/testutil/rest => github.com/cosmos/cosmos-sdk v0.46.0-alpha2
+	github.com/cosmos/cosmos-sdk/x/bank/testutil => github.com/cosmos/cosmos-sdk v0.46.0-alpha2
+	github.com/tendermint/tendermint/libs/time => github.com/tendermint/tendermint v0.35.0
```

Comment:
These lines were added to attempt to solve the error in the go.mod file, but they were unsuccessful. The issue is the same I was having in the last PR that I made [comments](https://github.com/pomifer/chihuahua/pull/4#discussion_r798979668) about.

Reply:
I think we already discussed this, but for posterity, go modules doesn't work like this, you have to replace entire repos we can't replace only packages